content/docs/command-reference/remote/modify.md

```diff
@@ -300,12 +297,16 @@ a full guide on using Google Drive as DVC remote storage.
   access Google Drive data. `.dvc/tmp/gdrive-user-credentials.json` by default.
 
   ```dvc
-  $ dvc remote modify myremote gdrive_user_credentials_file \
+  $ dvc remote modify --local myremote gdrive_user_credentials_file \
```

Comment:
this is safe to share (it's just a path)

Reply:
Rolled back.

> I realize it's a `path`. But since it's a path to a secret file, secure projects would hide this, as not to publish any hints for a malicious user on what kind of system is being used or where to look. I also said "...may contain sensitive..." in the note for this one. Anyway...